64_FFT/app_resources/common.hlsl

```diff
@@ -0,0 +1,23 @@
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+// Unfortunately not every piece of C++14 metaprogramming syntax is available in HLSL 202x
+// https://github.com/microsoft/DirectXShaderCompiler/issues/5751#issuecomment-1800847954
+typedef nbl::hlsl::float32_t2 input_t;
+typedef nbl::hlsl::float32_t output_t;
+
+NBL_CONSTEXPR_STATIC_INLINE uint32_t MaxPossibleElementCount = 1 << 20;
+
+struct PushConstantData
+{
+	uint64_t inputAddress;
+	uint64_t outputAddress;
+	uint32_t dataElementCount;
+};
+
+NBL_CONSTEXPR uint32_t WorkgroupSize = 256;
+NBL_CONSTEXPR uint32_t SubgroupSize = 32;
```

Comment:
you shall not hardcode the subgroup size!!!! it comes from the JIT device traits!

Reply:
how can I access those?

Reply:
WorkgroupSize is obviously hardcoded because you set that.

But SubgroupSize is not known ahead of time (not unless you use SubgroupSize Control on the Pipeline Creation) so its a builtin (non-constexpr) constant here:
https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/include/nbl/builtin/hlsl/glsl_compat/subgroup_basic.hlsl#L19

Reply:
I fixed it for you.